<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavePresetDialog</class>
 <widget class="QDialog" name="SavePresetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>109</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save preset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Do you want to save the preset?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="newTitlePresent">
     <property name="text">
      <string>&amp;Save as...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>newTitle</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newTitle">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newTitlePresent</tabstop>
  <tabstop>newTitle</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SavePresetDialog</receiver>
   <slot>onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SavePresetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newTitlePresent</sender>
   <signal>toggled(bool)</signal>
   <receiver>SavePresetDialog</receiver>
   <slot>onEnablePresetNameUI(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SavePresetDialog</receiver>
   <slot>onChangedPresetName(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEnablePresetNameUI(bool)</slot>
  <slot>onOk()</slot>
  <slot>onChangedPresetName(QString)</slot>
 </slots>
</ui>
